lib/tasks/generate_random_data.rake

```diff
@@ -0,0 +1,26 @@
+namespace :generate_random_data do
```

Comment:
A bit more specific name would be great i guess? something like "generate_users_data"

Reply:
I had the intention of using this generic namespace for populating tables with random data as per requirement. The users table is just one of the many, and so if required at some later stage we can add more tasks in this file to populate other tables as well.
Do you want me to make this file specific to users only ?

Reply:
@arunava59 oh yeah, great then.